tests/integration/mounts.bats

```diff
@@ -69,7 +69,7 @@ function teardown() {
 	# Without cgroup namespace.
 	# shellcheck disable=SC2016
 	update_config '	  .linux.namespaces -= [{"type": "cgroup"}]
-			| .process.args |= ["sh", "-euc", "for f in `grep /sys/fs/cgroup /proc/mounts | awk \"{print \\\\$2}\"| uniq`; do grep -w $f /proc/mounts | tail -n1; done"]'
+			| .process.args |= ["sh", "-euc", "for f in `grep /sys/fs/cgroup /proc/mounts | awk \"{print \\\\$2}\"| uniq`; do test -e $f && grep -w $f /proc/mounts | tail -n1; done"]'
```

Comment:
The alternative fix would be to actually mount `/sys/fs/cgroup/unified` into the container (and then remount it read-only). I'd rather not do that.